tests/v2/devfiles/components/pluginAll.yaml

```diff
@@ -0,0 +1,47 @@
+  - name: "testpluginuri"
+    plugin:
+      uri: "https://raw.githubusercontent.com/devfile/library/master/tests/v2/devfiles/samples/testParent.yaml"
```

Comment:
no kubernetes or id in snippet, but look out for the PR to remove plugin. so we may need to delete these tests

Reply:
yes, parser currently rejects anything other than uri so I took kubernetes and id out. 
This error from parser: "plugin URI undefined, currently only URI is suppported"